src/app.js

```diff
@@ -1,21 +1,38 @@
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
-}
-
-module.exports = sum;
+const { createServer } = require('http');
+const { URL: NodeURL } = require('url');
+const fs = require('fs/promises');
+const path = require('path');
+
+createServer(async(req, res) => {
+  res.setHeader('Content-type', 'application/json');
+
+  const newUrl = new NodeURL(req.url, `http://${req.headers.host}`);
+  const pathWithoutFile = newUrl.pathname.replace('/file/', '');
+  const publicFolderPath = path.resolve(__dirname, '..', 'public');
+
+  if (pathWithoutFile.length && newUrl.pathname.slice(0, 6) === '/file/') {
+    try {
+      const fullPath = path.join(publicFolderPath, pathWithoutFile);
+      const content = await fs.readFile(fullPath);
+
+      res.end(content.toString());
+
+      return;
+    } catch (error) {
+      const nameOfFile = pathWithoutFile
+        .slice(pathWithoutFile.lastIndexOf('/') + 1);
+
+      res.writeHead(404, 'Not found');
```

Comment:
'res.writeHead' function takes a status code as the first argument and an optional status message as the second. You are passing a string 'Not found' as the second argument, which is not a valid status message. You should remove this argument.